test/common/tracing/lightstep_tracer_impl_test.cc

```diff
@@ -166,10 +166,16 @@ TEST_F(LightStepDriverTest, FlushSeveralSpans) {
       .WillOnce(Return(5000U));
 
   SpanPtr first_span = driver_->startSpan(request_headers_, operation_name_, start_time_);
-  first_span->finishSpan();
+  // MockFinalizer finalizer;
+  NullFinalizer finalizer;
```

Comment:
why not MockFinalizer, we could check later on that finalize is called on this mock